apps/ledger-live-desktop/src/renderer/components/SyncOnboarding/Manual/index.tsx

```diff
@@ -172,9 +176,13 @@ const SyncOnboardingManual = ({ deviceModelId: strDeviceModelId }: SyncOnboardin
         status: "inactive",
         title: "Nano applications",
         renderBody: () => (
-          <ApplicationContent
+          <OnboardingAppInstallStep
+            device={device}
+            restoreDevice={
+              shouldRestoreApps && restoreDevice ? (restoreDevice as DeviceModelInfo) : undefined
```

Comment:
Instead of using `as` here you could consider fixing the issue at the source: the `lastSeenDeviceSelector` function. 
Just add a type annotation to specify that function's signature there.

Reply:
Done